src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -320,11 +326,11 @@ private String getEditedAtText(Boolean isGiverAnonymous, String displayGiverAs,
 
     public String getEditedAtTextForInstructor(Boolean isGiverAnonymous) {
         return getEditedAtText(isGiverAnonymous, this.lastEditorEmail,
-                TimeHelper.formatTime(this.lastEditedAt));
+                sdf.format(this.lastEditedAt));
     }
 
     public String getEditedAtTextForStudent(Boolean isGiverAnonymous, String displayGiverAs) {
         return getEditedAtText(isGiverAnonymous, displayGiverAs,
-                TimeHelper.formatDate(this.lastEditedAt));
+                sdf.format(this.lastEditedAt));
     }
```

Comment:
Since we're standardizing everything, you may want to collapse everything to just `getEditedAtText`.